src/main/resources/avro/metadata.avdl

```diff
@@ -38,57 +38,57 @@ record Experiment {
   Granularity here is variable (e.g. date only).
   Format: ISO 8601, YYYY-MM-DDTHH:MM:SS (e.g. 2015-02-10T00:03:42)
   */
-  union { null, string } runTime = null;
+  string runTime;
 
   /**
   The molecule examined in this experiment. (e.g. genomics DNA, total RNA)
   */
-  union { null, string } molecule = null;
+  string molecule;
 
   /**
   The experiment technique or strategy applied to the sample.
   (e.g. whole genome sequencing, RNA-seq, RIP-seq)
   */
-  union { null, string } strategy = null;
+  string strategy;
 
   /**
   The method used to enrich the target. (e.g. immunoprecipitation, size
   fractionation, MNase digestion)
   */
-  union { null, string } selection = null;
+  string selection;
 
   /** The name of the library used as part of this experiment. */
-  union { null, string } library = null;
+  string library;
 
   /** The configuration of sequenced reads. (e.g. Single or Paired) */
-  union { null, string } libraryLayout = null;
+  string libraryLayout;
 
   /**
     The instrument model used as part of this experiment.
     This maps to sequencing technology in BAM.
   */
-  union { null, string } instrumentModel;
+  string instrumentModel;
 
   /**
   The data file generated by the instrument.
   TODO: This isn't actually a file is it?
   Should this be `instrumentData` instead?
   */
-  union { null, string } instrumentDataFile = null;
+  string instrumentDataFile;
 
   /** The sequencing center used as part of this experiment. */
-  union { null, string } sequencingCenter;
+  string sequencingCenter;
 
   /**
   The platform unit used as part of this experiment. This is a flowcell-barcode
   or slide unique identifier.
   */
-  union { null, string } platformUnit = null;
+  string platformUnit;
 
   /**
   A map of additional experiment information.
   */
-  map<array<string>> info = {};
+  map<array<string>> info;
```

Comment:
Similarly, ``info`` should not now be required --- I think the correct semantics is for it to remain
```
map<array<string>> info = {};
```